crates/core/src/constraint_system/prove.rs

```diff
@@ -393,6 +404,12 @@ where
 
 	let univariate_challenge = univariate_output.univariate_challenge;
 
+	let zerocheck_eq_ind_sumcheck_span = tracing::info_span!(
+		"[phase] Eq Ind Sumcheck",
```

Comment:
Can we label this "step" instead of "phase"? Reading "[phase] Eq Ind Sumcheck" makes it sound like a phase, but below it says the phase is zerocheck.

Reply:
Also, we call this "MLE-check" now, instead of "Eq Ind Sumcheck". (Diagram not updated yet)

Reply:
done both.